<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BPMOptionsPage</class>
 <widget class="QWidget" name="BPMOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>349</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="bpm_options">
     <property name="title">
      <string>BPM Analyze Parameters:</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>win_s:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="win_s_parameter"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>hop_s:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hop_s_parameter"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>samplerate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="samplerate_parameter"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>        # super fast
        samplerate, win_s, hop_s = 4000, 128, 64 
        # fast
        samplerate, win_s, hop_s = 8000, 512, 128
        # default:
        samplerate, win_s, hop_s = 44100, 1024, 512</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>263</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
